src/middlewared/middlewared/plugins/failover.py

```diff
@@ -90,6 +90,12 @@ async def failover_extend(self, data):
         Bool('master', null=True),
         update=True,
     ))
+    @returns(Dict(
+        Int('id'),
+        Bool('disabled'),
+        Int('timeout'),
```

Comment:
I suppose these are always going to be present ? We should add `required=True` in that case